src/query/plan/operator.cpp

```diff
@@ -26,6 +26,8 @@
 #include <cppitertools/chain.hpp>
 #include <cppitertools/imap.hpp>
 
+#include "auth/models.hpp"
```

Comment:
`mg-auth` is not a dependency of `mg-query`, thus you cannot include anything from auth.